feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,232 @@
+
+### DP-1.17.10 Egress Classification and rewrite of IPoGUE traffic with pop action
+*   Configuration:ƒ
+    *    Configure decapsulation rules for IPv4-DST-DECAP/32
+    *    Configufe egress QOS re-write rules based on DSCP/TC values and set new DSCP values based on remarking rules.
+
+*   Traffic:
+    *   Generate IPoGUE traffic from ATE Port 1 with payload reachable via ATE:Port2
+*   Verfication:
+    *   Monitor telemetry on  DUT ATE Port 1 to verify packet scheduling into correct egress QOS queues based on the DSCP markings of the payload prir to re-write rules by observing queue counters per configued QOS classes.. ƒ
+    *   Capture packets on the DUT ATE Port 2 ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.11 Egress Classification and rewrite of IPv6oGUE traffic with pop action
+*   Configuration:
+    *   Configure decapsulation rules for Id IPv6-DST-DECAP/12
+    *   Configufe egress QOS re-write rules based on DSCP/TC values and set new DSCP values based on remarking rules.
+
+*   Traffic:
+    *   Generate IPv6oGUE traffic from ATE Port 1 with IPv6 payload reachable via ATE:Port2
+*   Verfication:
+    *   Monitor telemetry on  DUT ATE Port 1 to verify packet scheduling into correct egress QOS queues based on the DSCP markings of the payload prir to re-write rules by observing queue counters per configued QOS classes. 
+    *   Capture packets on the DUT ATE Port 2 ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test.
+
+```yaml
+paths:
+  ## Config paths
+  /qos/classifiers/classifier/config/name:
+  /qos/classifiers/classifier/config/type:
+  /qos/classifiers/classifier/terms/term/config/id:
+  /qos/classifiers/classifier/terms/term/actions/config/target-group:
+  /qos/classifiers/classifier/terms/term/conditions/ipv4/config/dscp:
+  /qos/classifiers/classifier/terms/term/conditions/ipv4/config/dscp-set:
+  /qos/classifiers/classifier/terms/term/conditions/ipv6/config/dscp:
+  /qos/classifiers/classifier/terms/term/conditions/ipv6/config/dscp-set:
+  /qos/classifiers/classifier/terms/term/conditions/mpls/config/traffic-class:
+  /qos/classifiers/classifier/terms/term/actions/remark/config/set-dscp:
+  /qos/classifiers/classifier/terms/term/actions/remark/config/set-mpls-tc:
+  /qos/interfaces/interface/input/classifiers/classifier/config/name:
+  /qos/interfaces/interface/input/classifiers/classifier/config/type:
+
+  ## State paths
+  /qos/interfaces/interface/input/classifiers/classifier/terms/term/state/matched-packets:
+  /qos/interfaces/interface/input/classifiers/classifier/terms/term/state/matched-octets:
```

Comment:
You can also check 
  - /qos/interfaces/interface/output/queues/queue/state/dropped-pkts